addons/website_sale/models/website.py

```diff
@@ -186,7 +186,7 @@ def _get_pl_partner_order(
             pricelists |= partner_pricelist
 
         # This method is cached, must not return records! See also #8795
-        return pricelists.ids
+        return pricelists.sudo().sorted().ids
```

Comment:
Maybe add a comment to explain why the sudo is needed here ? (i suppose it's to bypass security rules in the sorted ?)